frontend/test/metabase/visualizations/components/settings/ChartSettingOrderedColumns.unit.spec.js

```diff
@@ -18,61 +16,70 @@ function renderChartSettingOrderedColumns(props) {
 
 describe("ChartSettingOrderedColumns", () => {
   it("should have the correct add and remove buttons", () => {
-    const setting = renderChartSettingOrderedColumns({
+    renderChartSettingOrderedColumns({
       value: [{ name: "Foo", enabled: true }, { name: "Bar", enabled: false }],
     });
-    expect(setting.find(".Icon-add")).toHaveLength(1);
-    expect(setting.find(".Icon-close")).toHaveLength(1);
+    screen.getByRole("img", { name: /add/i });
+    screen.getByRole("img", { name: /close/i });
   });
+
   it("should add a column", () => {
     const onChange = jest.fn();
-    const setting = renderChartSettingOrderedColumns({
+    renderChartSettingOrderedColumns({
       value: [{ name: "Foo", enabled: true }, { name: "Bar", enabled: false }],
       onChange,
     });
-    setting.find(".Icon-add").simulate("click");
+    const ADD = screen.getByRole("img", { name: /add/i });
+
+    fireEvent.click(ADD);
     expect(onChange.mock.calls).toEqual([
       [[{ name: "Foo", enabled: true }, { name: "Bar", enabled: true }]],
     ]);
   });
+
   it("should remove a column", () => {
     const onChange = jest.fn();
-    const setting = renderChartSettingOrderedColumns({
+    renderChartSettingOrderedColumns({
       value: [{ name: "Foo", enabled: true }, { name: "Bar", enabled: false }],
       onChange,
     });
-    setting.find(".Icon-close").simulate("click");
+    const CLOSE = screen.getByRole("img", { name: /close/i });
+
+    fireEvent.click(CLOSE);
     expect(onChange.mock.calls).toEqual([
       [[{ name: "Foo", enabled: false }, { name: "Bar", enabled: false }]],
     ]);
   });
-  it("should reorder columns", () => {
-    const onChange = jest.fn();
-    const setting = renderChartSettingOrderedColumns({
-      value: [{ name: "Foo", enabled: true }, { name: "Bar", enabled: true }],
-      onChange,
-    });
-    // just call handleSortEnd directly for now as it's difficult to simulate drag and drop
-    setting.instance().handleSortEnd({ oldIndex: 1, newIndex: 0 });
-    expect(onChange.mock.calls).toEqual([
-      [[{ name: "Bar", enabled: true }, { name: "Foo", enabled: true }]],
-    ]);
+
+  // TODO: Test this in Cypress
```

Comment:
👍 on moving this to Cypress. That both allows a more real drag-and-drop, but also reordering columns requires a re-run to actually display the new table.